client/post-editor/editor-ground-control/save-status.jsx

```diff
@@ -0,0 +1,56 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { get, noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import postUtils from 'lib/posts/utils';
+
+// First, validate that the post had an ID, presumably this
```

Comment:
I forgot to mention that these comments will ofc be removed before merge. They're here to ask your opinion though - I don't know that `isPostNotYetPublished` is the best naming here